### ISSUE TITLE
Is pull after push supposed to give an error?

### ISSUE THREAD

**Issue by hex-profile** [ingydotnet/git-subrepo#487]
For example:

<Create an empty repository with initial commit.>

>git subrepo clone https://github.com/hex-profile/hexlib.git hexlib
Subrepo 'https://github.com/hex-profile/hexlib.git' (master) cloned into 'hexlib'.

>git subrepo pull hexlib
Subrepo 'hexlib' is up to date.

>git subrepo push hexlib
Subrepo 'hexlib' has no new commits to push.

>git subrepo pull hexlib
git-subrepo: There is already a worktree with branch subrepo/hexlib.
Use the --force flag to override this check or perform a subrepo clean
to remove the worktree.

**Comment by admorgan** [comment]
No, this is a bug. I think I have already found the source of the bug, but I haven't had a free minute to address it.

**Comment by hex-profile** [comment]
It's on version 0.4.1 (windows 64-bit), maybe I need to update?

**Comment by admorgan** [comment]
This is still an issue on the 0.4.2 branch, although there are a lot of bug fixes on that branch you might want to utilize.
